### PR TITLE
Added info on Dashboard support for split graph

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do?

Add information for Dashboard supposrt for Split Graph feature
Esther Test Dashboard Datadog 2023-04-06 at 11 44 32 AM

### Motivation

[DOCS-5091](https://datadoghq.atlassian.net/browse/DOCS-5091)

### Additional Notes

Ready to merge, PM reviewed and approved release.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-5091]: https://datadoghq.atlassian.net/browse/DOCS-5091?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by omar-nema** [comment]
Hey Esther --

I'd clarify here that this is not specific to metrics, it works on all data sources ("you can see your metric visualizations broken out by tags.") Would recommend specifying that 1 visualization appears for each instance of that tag value

For item (1) -- I think you can just condense it and say that this can be accessed through Split Graph tab when viewing graphs

I don't think (4) is necessary - instructions on changing the vis type

Overall this querying page feels too long to me. I don't think this 'configuring an APM stats graph' section is needed - I suggest breaking that out into its own page

File: content/en/dashboards/querying/_index.md [hunk 1]
```diff
@@ -220,7 +222,16 @@ With the event overlay, you can quickly see how actions within the organization
 - Correlating egress traffic with suspicious login activity
 - Correlating any timeseries data with monitor alerts to ensure that Datadog has been configured with the appropriate alerts
 
-{{< img src="/dashboards/querying/event_overlay_example.png" alt="Timeseries widgets showing RUM error rates with deployment events overlaid" style="width:100%;" >}}
+
+### Split graph
+
+With split graph, you can see your metric visualizations broken out by tags.
```
Comment (cswatt) [inline content/en/dashboards/querying/_index.md:228]:
```suggestion
With split graphs, you can see your metric visualizations broken out by tags. 
```

File: content/en/dashboards/querying/_index.md [hunk 2]
```diff
@@ -220,7 +222,16 @@ With the event overlay, you can quickly see how actions within the organization
 - Correlating egress traffic with suspicious login activity
 - Correlating any timeseries data with monitor alerts to ensure that Datadog has been configured with the appropriate alerts
 
-{{< img src="/dashboards/querying/event_overlay_example.png" alt="Timeseries widgets showing RUM error rates with deployment events overlaid" style="width:100%;" >}}
+
+### Split graph
+
+With split graph, you can see your metric visualizations broken out by tags. 
+
+{{< img src="dashboards/querying/split_graph_beta.png" alt="View split graphs of metric container.cpu.usage in the widget fullscreen" style="width:100%;" >}}
```
Comment (cswatt) [inline content/en/dashboards/querying/_index.md:230]:
```suggestion
{{< img src="dashboards/querying/split_graph_beta.png" alt="View split graphs of metric container.cpu.usage in the fullscreen widget" style="width:100%;" >}}
```

**Review by cswatt — CHANGES_REQUESTED** [review]
tiny tiny nits

**Pull request merged**